docs/charts.md

```diff
@@ -846,6 +846,13 @@ remote repository servers. This is because doing so would add
 substantial requirements to an implementing server, and thus raise the
 barrier for setting up a repository.
 
+### Chart Museum
```

Comment:
Would it read better to be incorporated into the Chart Repositories section rather than in a subsection below it?